integreat_cms/release_notes/current/unreleased/2793.yml

```diff
@@ -0,0 +1,2 @@
+en: New devtool to update testdata
+de: Neues Developertool zum Aktualisieren der Testdaten
```

Comment:
I believe we should skip the release note for this pull request. The developers will likely see the pull request directly, and including a note might confuse the users. 😄

Reply:
Ah, you are right. 😄 Thanks!

Reply:
THis has reappeared ^^

Reply:
Thanks :D